package.json

```diff
@@ -2,78 +2,100 @@
-    "babel-loader": "^6.4.1",
-    "babel-plugin-angularjs-annotate": "^0.7.0",
-    "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "^1.2.2",
-    "copy-webpack-plugin": "^4.0.1",
-    "css-loader": "^0.27.3",
-    "eslint": "^3.3.0",
-    "eslint-config-angular": "^0.5.0",
-    "eslint-loader": "^1.7.0",
-    "eslint-plugin-angular": "^1.3.0",
-    "eslint-plugin-jasmine": "^1.8.1",
-    "extract-text-webpack-plugin": "^2.1.0",
-    "html-loader": "^0.4.5",
-    "istanbul-instrumenter-loader": "^2.0.0",
-    "jasmine-core": "^2.5.2",
-    "jasmine-promise-matchers": "^2.2.0",
-    "karma": "^1.5.0",
-    "karma-chrome-launcher": "^2.0.0",
-    "karma-coverage-istanbul-reporter": "^1.0.0",
-    "karma-jasmine": "^1.1.0",
-    "karma-mocha-reporter": "^2.0.4",
-    "karma-phantomjs-launcher": "^1.0.4",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^2.0.3",
-    "ng-annotate-loader": "^0.2.0",
-    "ngtemplate-loader": "^1.3.1",
-    "node-sass": "4.5.3",
-    "phantomjs-polyfill": "0.0.2",
-    "phantomjs-prebuilt": "^2.1.14",
-    "sass-loader": "^6.0.3",
-    "style-loader": "^0.16.0",
-    "webpack": "^2.3.3",
-    "webpack-bundle-analyzer": "^2.3.1",
-    "webpack-dev-server": "^2.4.5"
+    "@babel/core": "^7.5.5",
+    "@babel/plugin-transform-runtime": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/runtime": "^7.4.5",
+    "angular-mocks": "^1.7.8",
+    "babel-eslint": "^10.0.2",
+    "babel-jest": "^24.8.0",
+    "babel-loader": "^8.0.6",
+    "babel-plugin-angularjs-annotate": "^0.10.0",
+    "copy-webpack-plugin": "^5.0.3",
+    "css-loader": "^3.0.0",
+    "html-loader": "^0.5.5",
+    "jest": "^24.8.0",
+    "jest-date-mock": "^1.0.7",
+    "mini-css-extract-plugin": "^0.7.0",
+    "ng-annotate-loader": "^0.6.1",
+    "ngtemplate-loader": "^2.0.1",
+    "node-sass": "4.12.0",
+    "sass-loader": "^7.1.0",
+    "standard": "^13.1.0",
+    "style-loader": "^0.23.1",
+    "webpack": "^4.35.2",
+    "webpack-bundle-analyzer": "^3.3.2",
+    "webpack-cli": "^3.3.5",
+    "webpack-dev-server": "^3.7.2"
   },
   "scripts": {
     "start": "webpack-dev-server",
     "build": "webpack -p",
     "build:analyze": "webpack -p --env.analyze",
-    "test": "karma start",
-    "lint": "eslint 'src/**/*.js'"
+    "test": "jest",
+    "lint": "standard"
+  },
+  "jest": {
+    "collectCoverage": true,
+    "collectCoverageFrom": [
+      "src/**/*.js",
+      "!**/*.fixture.js"
+    ],
+    "restoreMocks": true,
+    "setupFilesAfterEnv": [
+      "angular",
+      "angular-mocks",
+      "jest-date-mock",
+      "<rootDir>/jest/setup.js"
+    ],
+    "moduleNameMapper": {
+      "^.+\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js"
+    },
+    "modulePaths": [
+      "<rootDir>/src"
+    ],
+    "transform": {
+      "^.+\\.js?$": "babel-jest",
+      "^.+\\.html$": "<rootDir>/jest/htmlTransform.js"
+    },
+    "verbose": true
+  },
+  "standard": {
+    "env": {
+      "jest": true
```

Comment:
Should something for the browser be included here? ESLint lets you specify different envs for different file globs. https://github.com/CruGlobal/missionhub-web/blob/master/.eslintrc.json#L10-L15

Reply:
Only browser global it complained of was `localStorage`, so I didn't think it was necessary.